build.sbt

```diff
@@ -170,36 +170,43 @@ lazy val packagingSettings = Seq(
 
   /* Docker config (http://sbt-native-packager.readthedocs.io/en/latest/formats/docker.html)
    */
-  dockerBaseImage := Dependency.V.OpenJDK,
```

Comment:
We probably don't need the dependency anymore?

Reply:
Yup. Removed 👍